tests/test_client.py

```diff
@@ -742,42 +742,6 @@ def test_analyse_html(client, requests_mock):
     assert response[0]["coveredText"] == "Appendizitis"
 
 
-def test_analyse_html(client, requests_mock):
```

Comment:
Why was this test removed?

Reply:
seemed to be the same test as in line 709

Reply:
Indeed. Good catch.